13_6_Closing_a_Channel.md

```diff
@@ -0,0 +1,210 @@
+
+#### Cooperative Close
+
+In this case both channel participants agree to close the channel and settle the final state to the blockchain. Both participants must be online and it's performed by broadcasting an unconditional spend of the funding transaction with an output to each peer. 
+
+#### Force Close
+
+In this case when only one participant is online or if the participants disagree on the last state of the channel,  so one peer can perform an unilateral close of the channel without the cooperation of the other node.   It's performed by broadcasting a commitment transaction that commits to a previous channel state which both parts have agreed upon.
+This commitment transaction contains the channel state divided in two parts: the balance of each participant and all the pending payments (HTLCs).
+
+### Node Information
+
+Now we'll show you how to get information about your channel using `lightning-cli listchannels` command.  The listchannels RPC command returns data on channels that are known to the node. Because channels may be bidirectional, up to 2 objects will be returned for each channel (one for each direction).   To query information about own channels we'll use jq tool showed in previous chapters.
+
+First we'll get our own node id public_key in NODEID variable.
+
+```
+c$ NODEID=$(lightning-cli --network=testnet getinfo | jq .id)
+c$ echo $NODEID
+"03fce2a20393a65b9d6cab5425f4cd33ddc621ade458efd69d652917e2b5eaf59c"
+c$
+```
+Later we'll use select to show only data containing public_key id as source or destination.
+
+```
+$ lightning-cli listchannels | jq '.channels[] | select(.source == '$NODEID' or .destination == '$NODEID')'
+{
+  "source": "03fce2a20393a65b9d6cab5425f4cd33ddc621ade458efd69d652917e2b5eaf59c",
+  "destination": "0302d48972ba7eef8b40696102ad114090fd4c146e381f18c7932a2a1d73566f84",
+  "short_channel_id": "1780768x12x1",
+  "public": true,
+  "satoshis": 280000,
+  "amount_msat": "280000000msat",
+  "message_flags": 1,
+  "channel_flags": 2,
+  "active": true,
+  "last_update": 1595508075,
+  "base_fee_millisatoshi": 1000,
+  "fee_per_millionth": 1,
+  "delay": 40,
+  "htlc_minimum_msat": "1000msat",
+  "htlc_maximum_msat": "280000000msat",
+  "features": ""
+}
+```
+### Closing a channel
+
+Finally you should use `lightning-cli close` command to close the channel. The close RPC command attempts to close the channel cooperatively with the peer,  if you want to close it unilaterally set unilateraltimeout argument with number of seconds command will wait.   If you set to 0 and the peer is online command can negotiate a mutual close.   For this example we use an mutual close.
+
+```
+lightning-cli --network=testnet close 0302d48972ba7eef8b40696102ad114090fd4c146e381f18c7932a2a1d73566f84 0
```

Comment:
General practice is to capture that long ID if possible. Is it always the destination? If so, we should say that and capture it. (We can always show the whole info, like you do above, then afterward show a capture of the required variable.)